utils.php

```diff
@@ -30,7 +30,7 @@ public static function cache_bust_mtime( $path, $siteurl ) {
 			$dependency_path_mapping = new Page_Optimize_Dependency_Path_Mapping();
 		}
 
-		$file = $dependency_path_mapping->uri_path_to_fs_path( $url );
+		$file = $dependency_path_mapping->dependency_src_to_fs_path( $url );
```

Comment:
This also works if you pass `$path` to `uri_path_to_fs_path()`, but I think it may be slightly better to do what is in this PR because it is explicitly based on the site URLcoming from WP_Scripts or WP_Styles `$this->base_url`. Otherwise, path resolution is based on the default site URL determined by Page_Optimize_Dependency_Path_Mapping. In practice, it probably doesn't matter, but I like respecting what WP_Scripts and WP_Styles provide.